src/templates/static/platforms.jsx

```diff
@@ -15,29 +15,38 @@ const Platforms = ({
     description,
     text,
     status,
-    url = '',
-    target,
-    button_text,
-    download = '',
-}) => (
-    <div className={`gr-row gr-padding-30 ${className || ''}`} data-show={data_show}>
-        <div className='gr-4 gr-12-m gr-12-p gr-no-gutter-left gr-gutter-left-p gr-gutter-left-m center-text no-center-text-p-m'>
-            <img className='platform responsive' src={it.url_for(`images/pages/${image_path}/${image}.png`)} />
-        </div>
-        <div className='gr-8 gr-12-m gr-12-p'>
-            <h3 className={`section-title ${status === 'beta' ? '' : status || ''}`}>{header}{status === 'beta' ? <span className='beta'>{it.L('BETA')}</span> : ''}</h3>
-            <strong>{description}</strong>
-            <p>{text}</p>
-            <div className='gr-row'>
-                <div className='gr-12'>
-                    { url &&
-                        <a className='button-secondary' download={download || undefined} href={url} target={target || undefined} rel={/http/.test(url) ? 'noopener noreferrer' : undefined}><span>{button_text}</span></a>
-                    }
+    buttons,
+}) => {
+    const btnStyle = { display: 'inline-block', marginRight: '5px' };
```

Comment:
```suggestion
    const btn_style = { display: 'inline-block', marginRight: '5px' };
```

Reply:
it is only used once. why are we declaring it outside? it's changing the indentation of the whole block

Reply:
about indentation, any suggestion?

Reply:
I mean if we don't make a variable for it, we won't need to add `return (` so the indentations won't change

Reply:
ah OK :) I thought you're talking about the styles and their effect on the buttons.